### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ivan-kadykalo**
[DEMO LINK](https://ivan-kadykalo.github.io/react_add-todo-form/)

### CODE REVIEW

**Review by strukk311 — APPROVED** [review]
Wow, your styles are awesome 😍💥 Good job

**Review by MaksimusCRS — CHANGES_REQUESTED** [review]
I can add post without title 😕 

https://user-images.githubusercontent.com/92159085/168594332-7ec72994-12d6-4ce4-94fd-6b977a6bed59.mov

File: src/App.tsx [hunk 1]
```diff
@@ -1,17 +1,131 @@
...
+    setIsValidUserName(true);
+  };
+
+  const submitForm = (event: FormEvent<HTMLFormElement>) => {
+    if (title === InitialTodo.Title) {
+      setIsValidTitle(false);
+    } else if (userName === InitialTodo.UserName) {
+      setIsValidUserName(false);
+    } else if (isValidTitle && isValidUserName) {
+      const lastId = todos.length + 1;
```
Comment (MaksimusCRS) [inline src/App.tsx:47]: switch better here

**Review by MaksimusCRS — APPROVED** [review]
Looks super cool 🔥

**Pull request closed without merging**